packages/core/src/geometry/GeometrySystem.js

```diff
@@ -35,6 +35,8 @@ export default class GeometrySystem extends System
          * @readonly
          */
         this.hasInstance = true;
+
+        this.cache = {};
```

Comment:
Could you doc this cache property? Most important is private vs public for new props.